source/includes/steps-starting-vsce-connect-with-atlas-cli.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Given that this is a new content, I recommend using `.. procedure::` and `.. step::` directives in this procedure.

Spitballing:

```
.. procedure::
   :style: normal

   .. step:: Connect from the Atlas CLI. 
   
       In your terminal, run ...
```